File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.java
class Solution {
    public int largestAltitude(int[] gain) {
        
        
        int tempHight = 0;
        
        int res = 0;
        
        for(int g: gain) {
            
            tempHight+=g;
            
            res=Math.max(res, tempHight);
        }
        
        return res;
        
        
        
    }
}

File: 1732-find-the-highest-altitude/1732-find-the-highest-altitude.py
class Solution:
    def largestAltitude(self, gain: List[int]) -> int:
        
        
        temp_hight=0
        res = 0
        
        for i, g in enumerate(gain):
            
            temp_hight+=g
            
            res = max(temp_hight, res)
        
        return res
        